amd/src/ggbq.js

```diff
@@ -95,8 +108,24 @@ define(['jquery', '//www.geogebra.org/apps/deployggb.js'], function ($, GGBApple
                 parameters.ggbBase64 = this.ggbBase64[slot];
             }
 
-            parameters.scaleContainerClass = 'ggbscalingcontainer';
-            parameters.autoHeight = true;
+            // Check if width and height have been manually set. The default would be "no", so we use the scaling container feature.
+            if ((!ggbDataset.width || ggbDataset.width === '0') && (!ggbDataset.height || ggbDataset.height === '0')) {
+                parameters.scaleContainerClass = scalingContainers[slot];
+                parameters.autoHeight = true;
+            } else {
+                // Width and height are specified in this case, so we use the given fixed width and height settings
+                // of the plugin instance. Form validation of the settings asserts that both width and height are being set.
+                parameters.width = ggbDataset.width;
+                parameters.height = ggbDataset.height;
+                // We need to use getElementsByClassName because colons are not allowed for jquery and Vanilla JS querySelector.
+                const scalingContainer = document.getElementsByClassName(scalingContainers[slot])[0];
+                // We should always find this container, just check to be extra safe.
+                if (scalingContainer) {
+                    // Width of the scaling container is being set after the applet has been loaded. So no need to specify it here.
+                    scalingContainer.style.overflowX = 'auto';
+                    scalingContainer.style.overflowY = 'hidden';
```

Comment:
Why is overflowY set to `'hidden'`? This prohibits the vertical scrollbar.
Is it for preventing scrolling issues on small screens?

Reply:
In all cases I tested there was no need of it, because the containing container just grows to whatever height you set. If you have an example of parts of the applet being cut off, pls provide some.
Other reason is: I didn't really manage to make the scroll bar disappear in case of "nearly fitting it", so I disabled it completely